demaceo/whats-cookin
# Features and Refinements

### What's this PR do? **Lots**
- Add more assets (icons, food images)
- Add animation and color when clicking bookmark icon
- Add feature to add recipe to 'to cook later' user list
- Fix bug 034acda
- Adjust styling on homepage to prevent overflow disaster
- Add User methods to remove recipes from favorites and cook later 3938416
- Replace some `<img>` tags with `<svg>` as `<img>` was breaking styling 
cc36e4d
- Add search functionality and results page 
8a3cc95

### How should this be manually tested?

- Click on ribbon icon next to recipes to add recipe to 'cook later' list for that user. 
- List will only be updated if user is selected
- Logging in new user should refresh homepage with new random recipes in sidebar and staff picks

### Any background context you want to provide?

- `<img>` containers for the svg icons were creating a visible box around the icon elements. Swapping for the `<svg>` tag fixed it, but required styling changes and additional HTML elements and logic. a `<span>` was added with a `hidden` class and is injected with a recipe object 
- the DOM token list returned for `event.target.classList` did not allow `.includes()` as a function
